api/v1beta1/types_class.go

```diff
@@ -473,7 +473,7 @@ type SubnetClassSpec struct {
 	Name string `json:"name"`
 
 	// Role defines the subnet role (eg. Node, ControlPlane)
-	// +kubebuilder:validation:Enum=node;control-plane;bastion;all
+	// +kubebuilder:validation:Enum=node;control-plane;bastion;cluster
```

Comment:
I wonder if this is something we'd technically have to call a breaking change if `all` is no longer valid when it was before. Maybe we have more wiggle room here if `all` was never functional anyway?

Reply:
I thought of that--someone may have read the source code to figure out that "all" would pass validation. But AFAICT there is no code that handles `SubnetRole("all")`, no tests around that, and no constant defined for it, so it was not intentionally functional and didn't act as `SubnetCluster` is intended to.